src/groupby.jl

```diff
@@ -327,42 +352,61 @@ function DataAPI.groupby(
     end
     return groupby(A, dims)
 end
-function DataAPI.groupby(A::DimArrayOrStack, dimfuncs::DimTuple)
+function DataAPI.groupby(A::DimArrayOrStack, dimfuncs::DimTuple; name=:groupby)
```

Comment:
The name keyword here is not hit from normal use of the groupby function. I think we should add this keyword also to the other dispatches and forward it.

Reply:
I just added a PR to do this change.